interface/src/project/SettingsScheduler.tsx

```diff
@@ -52,15 +52,7 @@ import { useI18nContext } from 'i18n/i18n-react';
 import * as EMSESP from './api';
 
 function makeid() {
-  let result = '';
-  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-  const charactersLength = characters.length;
-  let counter = 0;
-  while (counter < 4) {
-    result += characters.charAt(Math.floor(Math.random() * charactersLength));
-    counter += 1;
-  }
-  return result;
+  return Math.floor(Math.random() * (Math.floor(200) - 100) + 100);
```

Comment:
This seems a bit strange to me: floor(200) is always 200, but i think it willalways generate a number between 100 and 200.

Reply:
yes I wanted a number between 100 and 200. Math.random() returns a floating point. I would probably work without the floor(200). I should test it in a small program and use 'node prog.js' to see the output.

Reply:
I think `Math.floor(Math.random() * 100 + 100)` is the same, or `Math.floor(Math.random() * 100) + 100`